app/javascript/mastodon/features/getting_started/index.js

```diff
@@ -57,7 +57,7 @@ class GettingStarted extends ImmutablePureComponent {
 
         <div className='scrollable optionally-scrollable' style={{ display: 'flex', flexDirection: 'column' }}>
           <div className='static-content getting-started'>
-            <p><FormattedMessage id='getting_started.open_source_notice' defaultMessage='Mastodon is open source software. You can contribute or report issues on GitHub at {github}. {apps}.' values={{ github: <a href="https://github.com/tootsuite/mastodon" target="_blank">tootsuite/mastodon</a>, apps: <a href="https://github.com/tootsuite/documentation/blob/master/Using-Mastodon/Apps.md" target="_blank"><FormattedMessage id='getting_started.apps' defaultMessage='Various apps are available' /></a> }} /></p>
+            <p><FormattedMessage id='getting_started.support' defaultMessage='{faq} • {userguide} • {apps}' values={{ faq: <a href="https://github.com/tootsuite/documentation/blob/master/Using-Mastodon/FAQ.md" target="_blank"><FormattedMessage id='getting_started.faq' defaultMessage='FAQ' /></a>, userguide: <a href="https://github.com/tootsuite/documentation/blob/master/Using-Mastodon/User-guide.md" target="_blank"><FormattedMessage id='getting_started.userguide' defaultMessage='User Guide' /></a>, apps: <a href="https://github.com/tootsuite/documentation/blob/master/Using-Mastodon/Apps.md" target="_blank"><FormattedMessage id='getting_started.appsshort' defaultMessage='Apps' /></a> }} /><br /><FormattedMessage id='getting_started.open_source_notice' defaultMessage='Mastodon is open source software. You can contribute or report issues on GitHub at {github}.' values={{ github: <a href="https://github.com/tootsuite/mastodon" target="_blank">tootsuite/mastodon</a> }} /></p>
```

Comment:
@jeroenpraat `yarn run manage:translations` can not be executed unless it is a new master.